realm/realm-jni/src/util.cpp

```diff
@@ -185,21 +185,6 @@ jclass GetClass(JNIEnv* env, const char* classStr)
     return myClass;
 }
 
-void jprint(JNIEnv *env, char *txt)
```

Comment:
I also saw this function was removed, but thought it was ok as we'd be able to put a breakpoint after @beeender's #2960 completed. Relying printf for JNI debugg has been pain in the neck, tbh.